lib/Platform/Intl/impl_icu/LocaleResolver.cpp

```diff
@@ -0,0 +1,306 @@
+/**
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+#include "LocaleResolver.h"
+
+#include "Constants.h"
+#include "IntlUtils.h"
+#include "Locale.h"
+#include "LocaleBCP47Object.h"
+#include "OptionHelpers.h"
+#include "hermes/Platform/Unicode/icu.h"
+
+namespace hermes {
+namespace platform_intl {
+namespace impl_icu {
+
+const LocaleResolver::AvailableLocale LocaleResolver::availableLocales_ =
```

Comment:
This introduces a global constructor. Please wrap it as a function-local static instead.

Reply:
Fixed in 3rd commit.